src/mongo/tree/MongoDocumentTreeItem.ts

```diff
@@ -63,6 +63,9 @@ export class MongoDocumentTreeItem implements IAzureTreeItem {
     }
 
     public static async update(collection: Collection, newDocument: IMongoDocument): Promise<IMongoDocument> {
+        if (!newDocument["_id"]) {
```

Comment:
Does this work if newDocument is null? Or do you need to do:
```
if (!newDocument || !newDocument["_id"])
```

Reply:
I tried the following : 
1. Empty document ({}) - In this case, `newDocument` exists, but `newDocument["_id"]` does not. 
2. Empty text  - it doesn't hit this line, since the JSON parser throwsan `Unexpected end of input` error